testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/naming/remote/simple/RemoteNamingHTTPTestCase.java

```diff
@@ -78,7 +77,9 @@ public  Context getRemoteHTTPContext() throws Exception {
         URI webUri = managementClient.getWebUri();
         URI namingUri = new URI("http", webUri.getUserInfo(), webUri.getHost(), webUri.getPort(), "/wildfly-services", "" ,"");
         env.put(Context.PROVIDER_URL, namingUri.toString());
-        return new InitialContext(DefaultConfiguration.addSecurityProperties(env));
+        env.put(Context.SECURITY_PRINCIPAL, System.getProperty("jboss.application.username", "guest"));
+        env.put(Context.SECURITY_CREDENTIALS, System.getProperty("jboss.application.username", "guest"));
```

Comment:
SECURITY_CREDENTIALS   ...  System.getProperty("jboss.application.username", ...) 

Shouldn't be there jboss.application.password ?